### ISSUE TITLE
[BUG] Styles from scss, are ignored

### ISSUE THREAD

**Issue by PoSayDone** [elkowar/eww#605]
### Checklist before submitting an issue

- [X] I have searched through the existing [closed and open issues](https://github.com/elkowar/eww/issues?q=is%3Aissue) for eww and made sure this is not a duplicate
- [X] I have specifically verified that this bug is not a common [user error](https://github.com/elkowar/eww/issues?q=is%3Aissue+label%3Ano-actual-bug+is%3Aclosed)
- [X] I am providing as much relevant information as I am able to in this bug report (Minimal config to reproduce the issue for example, if applicable)

### Description of the bug

Background style from yuck file is ignoring background style from scss file.

https://user-images.githubusercontent.com/29358657/197986377-bad11aa3-8102-44da-a2ee-ee0a15e15c6f.mp4

### Reproducing the issue

Create a box with background in yuck file and try to overwrite child blocks background from scss

### Expected behaviour

Use background color from scss

**Comment by PoSayDone** [comment]
I figured out something strange. It seems like it's not my title and artist blocks background

https://user-images.githubusercontent.com/29358657/197989770-fd12c3a3-a25e-4ed0-af0e-f5226dde8557.mp4

**Comment by PoSayDone** [comment]
If I set the rendering mode "Picture", the problem is solved. But how to set it permanent?

https://user-images.githubusercontent.com/29358657/198052759-bf4a748d-672b-440b-a1b6-1e1d532bc88c.mp4

**Comment by PoSayDone** [comment]
I realized that this is a rendering bug, because css has no effect on the appearance of these strange backgrounds. I also noticed that these backgrounds appear when the variables that use these elements are updated. 

https://user-images.githubusercontent.com/29358657/199301729-dd14ec6a-a691-4278-883b-3fef0c979644.mp4

https://user-images.githubusercontent.com/29358657/199301894-ae0e9c81-ad1d-4db2-b86d-ad59a0ef157e.mp4

Also there is no such glitches when using "Image" rendering mode, as I said before

**Comment by viandoxdev** [comment]
You sure you just didn't activate the show widget resize option of the inspector ? it makes any widget that changes size flash red

**Comment by PoSayDone** [comment]
> You sure you just didn't activate the show widget resize option of the inspector ? it makes any widget that updates flash red

Yes, I'm sure. Second video in previous message is with enabled "Show grafical update", I've enabled it to show that this glitches appears with similar timing as variables updates
